styleguide/derek/global-components/back-to-top/back-to-top.js

```diff
@@ -0,0 +1,8 @@
+// back to top
+const $ = jQuery;
+const duration = 300;
+$('.backtotop-link').click((event) => {
+  event.preventDefault();
+  $('html, body').animate({ scrollTop: 0 }, duration);
+  return false;
+});
```

Comment:
To be more modular can you wrap this in an iife?  Otherwise, it could rewrite $ for other things. 

```
(function($) {
  const duration = 300;
   $('.backtotop-link').click((event) => {
     event.preventDefault();
     $('html, body').animate({ scrollTop: 0 }, duration);
     return false;
   });
})(jQuery);
```